docs/_book.yaml

```diff
@@ -33,6 +33,18 @@ upper_tabs:
       - title: "Constructing Trotter Steps With Low-Rank Decomposition"
         path: /openfermion/tutorials/circuits_3_arbitrary_basis_trotter
 
+      - heading: "FQE Guide & Tutorials"
```

Comment:
I think the plan was to add sections to existing lower-tabs, yes? Similar to https://www.tensorflow.org/responsible_ai/guide

The RAI `_book.yaml` isn't available in an OSS repo but I can ping you the internal link, if you'd like.

Reply:
Talked with Nick and the conclusion was to use the current structure (with a different heading name).